packages/okidoc-md/src/buildMarkdown/buildMarkdown.spec.js

```diff
@@ -365,15 +377,16 @@ describe('buildMarkdown', () => {
 
     it('should render valid markdown with function in return type interface', async () => {
       const documentationSource = `
-        /**
-        * @property update - update entity
-        * @property destroy - destroy entity
-        */
+        /** */
         interface MyEntity {
+          /** */
           id: string;
+          /** */
           title: string;
-          updateTitle(id: string, title: string): Promise<any>;
-          destroy(): void;
```

Comment:
is not it also valid format?

Reply:
It is valid in TypeScript, but latest version of documentationjs fails to understand type of this member (as well as params).

Reply:
:(
Then mb we should prepare this part on our side:
```js
convert
updateTitle(id: string, title: string): Promise<any>;
to 
updateTitle: (id: string, title: string) => Promise<any>;
```

Reply:
That might be possible solution.